src/server/scripts/EasternKingdoms/Stratholme/boss_baron_rivendare.cpp

```diff
@@ -53,13 +53,19 @@ enum Spells
     SPELL_RAISE_DEAD6           = 17480,
 };
 
-// Define Add positions
-Position const ADD_POS_1 = {4017.403809f, -3339.703369f, 115.057655f, 5.487860f};
-Position const ADD_POS_2 = {4013.189209f, -3351.808350f, 115.052254f, 0.134280f};
-Position const ADD_POS_3 = {4017.738037f, -3363.478016f, 115.057274f, 0.723313f};
-Position const ADD_POS_4 = {4048.877197f, -3363.223633f, 115.054253f, 3.627735f};
-Position const ADD_POS_5 = {4051.777588f, -3350.893311f, 115.055351f, 3.066176f};
-Position const ADD_POS_6 = {4048.375977f, -3339.966309f, 115.055222f, 2.457497f};
+enum RivendareMisc
+{
+    EVENT_SHADOW_BOLT           = 0,
```

Comment:
Zero isn't a valid index for events, events scheduled with 0 as id are never executed